### ISSUE TITLE
Button in command bar doesn't have parent with required aria role

### ISSUE THREAD

**Issue by miduncan** [microsoft/fluentui#19442]
### Environment Information

- **Package version(s)**: 8.28.2
- **Browser and OS versions**: Edge, Windows 10

### Describe the issue:

Menu button in the command bar (buttons that contain a menu) are marked with a role of `menu-item` but does not have a parent with one of the required roles (`menubar` or `menu`)
![image](https://user-images.githubusercontent.com/55718965/129986913-347feee1-60e6-4fce-9de6-ae3099b83076.png)

See [https://accessibilityinsights.io/info-examples/web/aria-required-parent](aria-required-parent) for more info.

### Please provide a reproduction of the issue in a codepen:

https://developer.microsoft.com/en-us/fluentui#/controls/web/commandbar
"Command Bar With Split and Disabled Buttons" example in your docs

### Documentation describing expected behavior

See [https://accessibilityinsights.io/info-examples/web/aria-required-parent](aria-required-parent) for more info.

**Comment by gouttierre** [comment]
@miduncan - - Thanks for filing this issue with us. To set expectation our team will have a look at the issue but only once capacity allows.
  
@bsunderhus -  Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by smhigley** [comment]
dupe of #18484 -- the root cause is that SplitButton has nested menuitem roles. See this issue comment for why we're keeping the nested roles until we can introduce breaking changes to behavior in vNext: https://github.com/microsoft/fluentui/issues/18484#issuecomment-861723118
